src/components/ebay-donut-chart/components/ebay-chart-legend/index.marko

```diff
@@ -0,0 +1,45 @@
+import type { DonutLegendItem } from "../../component";
+export interface Input {
+    focusIndex: number;
+    items: DonutLegendItem[];
+    "on-focus": (index: number) => void;
+    "on-blur": () => void;
+}
+class {
+    handleFocus(id: number) {
+        this.emit("focus", id);
+    }
+
+    handleBlur() {
+        this.emit("blur");
+    }
+}
+
+<div class="ebay-donut-chart__legend">
+    <for|item, index| of=input.items>
+        <div
+            class=[
+                "ebay-donut-chart__legend-item",
+                input.focusIndex !== -1 &&
```

Comment:
Can we change this logic to be
`!(input.focusIndex === index) && "ebay-donut-chart__legend-item--inactive"`
I assume that index is not -1. If it can be, then maybe its fine as is

Reply:
Yea, using `-1` is what I would consider an "old school" way of managing the null state of the focus index haha

So in this instance, none of these should have the inactive if there's no `active` index

I can update this logic to be a little better `input.focusIndex !== null &&`